src/continuation/continuation_clustering.jl

```diff
@@ -0,0 +1,170 @@
+- `par_weight = 1` The distance matrix between features has a special extra  weight that 
+  is proportional to the distance |pi - pj| between the parameters of each features. This 
+  keyword argument is the weight coeficient that ponderates the distance matrix.
+- `ϵ_optimal = 1.` This is the optimal radius for the DBSCAN clustering algorithm. It should be set 
+  manually. 
+- `mmap_limit = 20000` this parameter sets the limit of features that should be clustered using only the 
+  RAM memory. Above this limit the program uses a memory map based array to store the distance 
+  matrix on the disk. 
+
+## Description
+
+The method first simulate and compute a set of statistics on the trajectories, for example the mean 
+ and standard deviation of the trajectory data points. Once all the featured statistics have been 
+ computed, the algorithm computes the distance between each feature. A special weitgh term is added 
+ so that the distance between features with different parameters is increased. It helps to discriminate
+ between attractors with very different parameters. At last, the distance matrix is clustered with the 
+ DBSCAN algorithm
+
+[^Gelbrecht2020]:
+    Gelbrecht, M., Kurths, J., & Hellmann, F. (2020). Monte Carlo basin bifurcation
+    analysis. New Journal of Physics, 22(3), 033032.
+
+[^Stender2021]:
+    Stender & Hoffmann, [bSTAB: an open-source software for computing the basin
+    stability of multi-stable dynamical systems](https://doi.org/10.1007/s11071-021-06786-5)
+"""
+function ClusteringAcrossParametersContinuation(
+        mapper::AttractorsViaFeaturizing;
+        info_extraction = mean_across_features
+        # TODO: Here we can add more keywords regarding how to cluster across parameters.
+    )
+    return (; mapper, info_extraction)
+end
+
+function mean_across_features(fs)
+    means = zeros(length(first(fs)))
+    N = length(fs)
+    for f in fs
+        for i in eachindex(f)
+            means[i] += f[i]
+        end
+    end
+    return means ./ N
+end
+
+
+
+function basins_fractions_continuation(
+        continuation::NamedTuple, prange, pidx, ics::Function;
+        samples_per_parameter = 100, show_progress = true, par_weight = 1, 
+        ϵ_optimal = 1., mmap_limit = 20000
+    )
+    spp, n = samples_per_parameter, length(prange)
+    (; mapper, info_extraction) = continuation
+    progress = ProgressMeter.Progress(n;
+        desc="Continuating basins fractions:", enabled=show_progress
+    )
+
+    # Extract the first possible feature to initialize the features container
+    feature = extract_features(mapper, ics; N = 1)
+    features = Vector{typeof(feature[1])}(undef, n*spp)
+    # Collect features
+    for (i, p) in enumerate(prange)
+        set_parameter!(mapper.integ, pidx, p)
+        current_features = extract_features_threaded(mapper, ics; show_progress, N = spp)
+        features[((i - 1)*spp + 1):i*spp] .= current_features
+        next!(progress)
+    end
```

Comment:
This must become its own function called something like `extract_features_across_parameters`. Furthermore, you should use `extract_features` instead of `extract_features_threaded`. The latter is called if `threaded = true`.